docs/tutorials/typescript/getting_started.md

```diff
@@ -0,0 +1,141 @@
+# NEAR AI Typescript SDK - Getting Started
+
+## Prerequisites
+
+Before creating a NEAR AI Typescript Agent, please make sure you have the [NEAR AI CLI](https://docs.near.ai/cli/) installed and have logged in with your Near wallet.
+
+- [ ] You have `Python 3.11.xx` in your machine. (**IMPORTANT** `Python 3.12+` currently not supported!)
+- [ ] Installing NEAR AI CLI `python3 -m pip install nearai`
+- [ ] Login to NEAR AI `nearai login`
+
+> You'll be provided with a URL to login with your NEAR account.
+>
+> Example:
+
+```shell
+$> nearai login
+
+Please visit the following URL to complete the login process: https://auth.near.ai?message=Welcome+to+NEAR+AI&nonce=<xyzxyzxyzxyzx>&recipient=ai.near&callbackUrl=http%3A%2F%2Flocalhost%3A63130%2Fcapture
+```
+
+- [ ] Login with NEAR by connecting with your wallet. After successfully logging in, you will see a confirmation screen. Close it and return to your terminal.
```

Comment:
You could prob skip this guide in favor of having them follow the CLI installation instructions above. That way we only have one place to manage installation instructions in docs.